servers/src/grin/seed.rs

```diff
@@ -172,15 +173,10 @@ fn monitor_peers(
 		total_count += 1;
 	}
 
-	let peers_count = peers.iter().connected().count();
-
-	let max_diff = peers.max_peer_difficulty();
-	let most_work_count = peers
-		.iter()
-		.outbound()
-		.with_difficulty(|x| x >= max_diff)
-		.connected()
-		.count();
+	let peers_iter = || peers.iter().connected();
+	let peers_count = peers_iter().count();
```

Comment:
The change to using empty closures here and in other files when iterating peers is just a rust convenience thing?

Reply:
We need to build a couple of instances of the same (or similar) iterator. 
And the closure just makes this a lot easier to keep them consistent.